packages/core/chrome/core-chrome-browser-internal/src/ui/project/navigation.tsx

```diff
@@ -6,49 +6,51 @@
  * Side Public License, v 1.
  */
 
-import React, { useCallback } from 'react';
-import useLocalStorage from 'react-use/lib/useLocalStorage';
 import { EuiButtonIcon, EuiCollapsibleNav, EuiThemeProvider, useEuiTheme } from '@elastic/eui';
+import React from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { Observable } from 'rxjs';
 
-const LOCAL_STORAGE_IS_OPEN_KEY = 'PROJECT_NAVIGATION_OPEN' as const;
+interface Props {
+  children: JSX.Element;
+  navIsOpen$: Observable<boolean>;
+  setNavIsOpen: (value: boolean) => void;
+}
 
-export const ProjectNavigation: React.FC = ({ children }) => {
-  const { euiTheme, colorMode } = useEuiTheme();
-
-  const [isOpen, setIsOpen] = useLocalStorage(LOCAL_STORAGE_IS_OPEN_KEY, true);
-
-  const toggleOpen = useCallback(() => {
-    setIsOpen(!isOpen);
-  }, [isOpen, setIsOpen]);
+export const ProjectNavigation: React.FC<Props> = ({ children, setNavIsOpen, ...observables }) => {
+  const { navIsOpen$ } = observables;
+  const { colorMode } = useEuiTheme();
+  const navIsOpen = useObservable(navIsOpen$);
 
   return (
-    <EuiThemeProvider colorMode={colorMode === 'DARK' ? 'LIGHT' : 'DARK'}>
```

Comment:
NOTE: Dark color mode is currently not working with our EUI Collapsible Nav Groups: https://github.com/elastic/eui/issues/6663